src/index.html

```diff
@@ -13,6 +13,52 @@
     />
```

Comment:
The attributes of the link tag should start on a new line with 2-space indentation related to the tag, and the tag's closing bracket should be on the same level as the opening one. This is to keep your attributes correctly formatted as per the HTML Formatting checklist.